contrib/helm/harbor/templates/ui/ui-dpl.yaml

```diff
@@ -38,8 +38,6 @@ spec:
             value: "1"
           - name: ADMINSERVER_URL
             value: "http://{{ template "harbor.fullname" . }}-adminserver"
-          - name: CLAIR_DB_HOST
```

Comment:
is this not needed in 1.5.0?

Reply:
Will call admin server to get the value.